cmd/cstor-pool-mgmt/cstorops/pool/pool.go

```diff
@@ -104,12 +124,27 @@ func GetPoolName() (string, error) {
 	return poolname, nil
 }
 
+// DeletePool destroys the pool created.
 func DeletePool(poolName string) error {
-	deletePoolStr := "zpool destroy -f " + poolName
+	deletePoolStr := PoolOperator + " destroy -f " + poolName
```

Comment:
why -f?

Reply:
why was it decided?

Reply:
Sometimes we got error saying unable to delete pool(couldn't recollect the exact stdout). @vishnuitta Can u pls recollect why we decided to go after -f?